src/hooks/usePersistentAspectRatio.ts

```diff
@@ -23,26 +23,31 @@ type Result = {
 };
 
 export default function usePersistentAspectRatio(url: string): Result {
+  if (url === null) url = 'default';
   const isSVG = isSupportedUriExtension(url, ['.svg']);
   const nonSvgUrl = isSVG ? imageToPng(url, 200) : url;
   const [ratio, setAspectRatio] = useMMKVNumber(nonSvgUrl as string, storage);
   const [state, setState] = useState<State>(
     ratio !== 0 ? State.loaded : State.init
   );
   useEffect(() => {
-    if (state === State.init && nonSvgUrl) {
+    if (state === State.init && nonSvgUrl && url !== 'default') {
       const lowResUrl = getLowResUrl(nonSvgUrl) as string;
       setState(State.loading);
-      Image.getSize(
-        lowResUrl,
-        (width, height) => {
-          setAspectRatio(width / height);
-          setState(State.loaded);
-        },
-        () => setState(State.failed)
-      );
+      lowResUrl !== null &&
```

Comment:
when `lowResUrl` is null?

Reply:
this I will remove

Reply:
I think this can be `undefined` tho. If signing will throw an exception. see `src/handlers/imgix.ts:91`